.scalafmt.conf

```diff
@@ -47,5 +47,5 @@ spaces {
 }
 
 project.excludeFilters = [
-  ".*\\.sc"
+  ".*\\.sc$"
```

Comment:
This was preventing the sources from being formatted on compile time. There is a bug in Scalafmt which read that reg exp incorrectly. Until they fix the bug, this is the workaround. We should also add, css, js, and html extensions here but I will do that in another PR.

Reply:
Great, finality it will work!